### ISSUE TITLE
Varmeta's Builder Journey

### ISSUE THREAD

**Issue by jamesnguyentech** [gnolang/hackerspace#40]
Welcome to Varmeta's Builder Journey!

This issue serves as a comprehensive introduction to Varmeta and a place where you can expect regular updates from us. We hope you'll find it very helpful to read this issue to learn more about Varmeta, and we look forward to collaborating with you! Let's embark on this journey together!

## Introduction:
Varmeta was founded in 2020 with a focus on blockchain and virtual reality/augmented reality technologies. Starting with only three members, we now have 40 engineers who are highly skilled tech experts, driven by determination, enthusiasm, and commitment to providing cutting-edge technology services.

We aim to contribute to the ecosystem of emerging blockchains, addressing lingering issues in traditional ones. We believe that the design philosophy for Gnoland developers has the potential to make it one of the most widely adopted smart contract platforms globally, and we are committed to supporting this initiative by providing various applications for its ecosystem.

Meet our team members:
Nha Tran - Project Manager - https://www.linkedin.com/in/nhatran38/
James Nguyen - Solution Architecture - https://www.linkedin.com/in/jamesnguyentech/
Truong Nguyen - Blockchain Developer - https://www.linkedin.com/in/ndtr2000/
Nhat Nguyen - Backend Developer - https://www.linkedin.com/in/thenhatat/
Anh Ngo - Backend Developer - https://www.linkedin.com/in/tuan-anh-ngo-052354262/
Chien Le - Backend Developer - https://www.linkedin.com/in/l%C3%AA-v%C4%83n-chi%E1%BA%BFn-184731186/
Hoang Pham - Frontend Developer
Hoai Le - Frontend Developer - https://www.linkedin.com/in/phuoc-hoai-0b0861172/
Vu Huynh - Frontend Developer - https://www.linkedin.com/in/vu-huynh-247023233/
Anh Pham - Tester - https://www.linkedin.com/in/anh-pham-39323b27b/
Trang Dang - Business Analyst - https://www.linkedin.com/in/%C4%91%E1%BA%B7ng-trang-66864ba6/

Varmeta Links:
Website: https://www.var-meta.com/
LinkedIn: https://www.linkedin.com/company/var-meta/

## Products:
I will provide you with brief explanations of our products with relevant links here.

### FanChain:
FanChain is a platform that enables streamers and influencers to create and issue NFT collections, including generative and membership NFTs. Fans can purchase these NFTs on our marketplace and participate in exclusive community events on Discord and Telegram. Additionally, FanChain offers a crypto-based donation system for viewers to support their favorite creators.

- **NFT Creation**: Streamers and influencers can create and issue NFT collections, including generative and membership NFTs, using ink! smart contracts.
- **Marketplace**: Users can buy, sell, and trade NFTs on our platform, fostering a vibrant and active community around content creators and their fans.
- **Community Management**: FanChain will offer tools for streamers and influencers to manage their Discord and Telegram communities, ensuring that only NFT holders can access exclusive channels and events. (Future plan)
- **Crypto Donations**: Viewers can support their favorite creators by donating cryptocurrency, with donation information displayed during live streams. (Future plan)
- **Compatibility**: FanChain will be compatible with all Wasm-supporting parachains, including prominent networks such as Astar Network, Phala Network, and more.

Technology Stack: FanChain will be built using ink! for smart contracts for the underlying blockchain, and a combination of React and Node.js for the frontend and backend.

Website: https://www.fanchain.network/
GitHub: (Currently private, will be made public in 2024)

We will use this hackerspace issue to provide regular updates on our journey in Gnoland, as well as the progress of our support initiatives. Lastly, the Varmeta team has enjoyed and appreciated working with the core members and contributors of Gnoland. We want to thank all of you for participating in this creative and promising ecosystem. Let's go!

**Comment by michelleellen** [comment]
Welcome @jamesnguyentech how can we get in touch, would love to have you join the developer call from time to time.

**Comment by jamesnguyentech** [comment]
Certainly! You can reach me at jamesnguyen@var-meta.com or feel free to DM me here. I'm excited to join the developer calls from time to time. Thanks for the invitation @michelleellen !
